README.md

```diff
@@ -364,3 +366,22 @@ project.afterEvaluate {
 
 Note that we have to wait for the evaluation phase to complete before the
 `generateReleasePlayResources` task becomes visible.
+
+## Advanced
+
+Options and configurations that are unusual, but available for advanced use-cases.
+
+### Alternate Build Folder
+
+If you use an external tool to process your APK file, you will need to override the standard
+publish process. With this enabled, the standard output will be ignored, and any `.apk` files
+in the provided folder will uploaded instead. If you have multiple flavors with different
+`applicationId`s you will need to use a property and override for each application you are
```

Comment:
I don't understand this sentence. There's only one property, what do you mean you need to set it for each?